app/config/collections.php

```diff
@@ -751,6 +751,107 @@
         ],
     ],
 
+    'schedules' => [
+        '$collection' => ID::custom(Database::METADATA),
+        '$id' => ID::custom('schedules'),
+        'name' => 'schedules',
+        'attributes' => [
+            [
+                '$id' => ID::custom('type'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => 100,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('scheduleId'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('projectId'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
+                'required' => false,
+                'default' => null,
+                'array' => false,
+                'filters' => [],
+            ],
+            [
+                '$id' => ID::custom('scheduleUpdatedAt'),
```

Comment:
```suggestion
                '$id' => ID::custom('resourceUpdatedAt'),
```

We can also group `resourceUpdatedAt` and `resourceId` below `projectId`